### ISSUE TITLE
Week 3 Memos: Uncertainty & Apocalypse

### ISSUE THREAD

**Issue by deholz** [deholz/AreWeDoomed24#6]
Reply with your memo as a Comment. The memo should be responsive to this week's readings from David Wallace-Wells, with 300–500 words + 1 visual element (e.g., figure, image, hand-drawn picture, art, etc. that complements or is suggestive of your argument). The memo should be tagged with one or more of the following:

#origin: How did we get here? Reflection on the historical, technological, political and other origins of this existential crisis that help us better understand and place it in context.

#risk: Qualitative and quantitative analysis of the risk associated with this challenge. This risk analysis could be locally in a particular place and time, or globally over a much longer period, in isolation or in relation to other existential challenges (e.g., the environmental devastation that follows nuclear fallout).

#policy: What individual and collective actions or policies could be (or have been) undertaken to avert the existential risk associated with this challenge? These could include a brief examination and evaluation of a historical context and policy (e.g., quarantining and plague), a comparison of existing policy options (e.g., cost-benefit analysis, ethical contrast), or design of a novel policy solution.

#solutions: Suggestions of what (else) might be done. These could be personal, technical, social, artistic, or anything that might reduce existential risk.

#framing: What are competing framings of this existential challenge? Are there any novel framings that could allow us to think about the challenge differently; that would make it more salient? How do different ethical, religious, political and other positions frame this challenge and its consequences (e.g., “End of the Times”).

#salience: Why is it hard to think and talk about or ultimately mobilize around this existential challenge? Are there agencies in society with an interest in downplaying the risks associated with this challenge? Are there ideologies that are inconsistent with this risk that make it hard to recognize or feel responsible for?

#nuclear/#climate/#bio/#cyber/#emerging: Partial list of topics of focus.

For one session over the course of the quarter, you may post a memo that reflects on a film or fictional rendering of an existential challenge. This should be tagged with:

#movie / #novel: How did the film/novel represent the existential challenge? What did this highlight; what did it ignore? How realistic was the risk? How salient (or insignificant) did it make the challenge for you? For others (e.g., from reviews, box office / retail receipts, or contemporary commentary)?

**Comment by DNT21711** [comment]
#Risk #Solutions

The existential threat of climate change, as depicted in the readings, reveals a complex interplay of immediate environmental catastrophes and potential for long-term ecological recovery. David Wallace-Wells' portrayal of a grim future is marked by escalating ecological disasters, painting a picture of a world where rising temperatures, melting ice caps, and extreme weather events are the new norm. These disasters are not isolated events but are interconnected, leading to a cascade of environmental and social consequences.

In contrast, Hannah Ritchie offers a counterpoint in "Not the End of the World," suggesting that significant strides have been made in sustainability. Her optimistic viewpoint highlights advancements in reducing carbon emissions and improving air quality, painting a more hopeful future. This dichotomy between Wallace-Wells' dire predictions and Ritchie's optimistic outlook underscores the complexity of climate risk. It suggests that while immediate action is critical to mitigate the worst impacts, there is also potential for recovery and adaptation in the long term.

Furthermore, the climate crisis is not just an environmental issue but a socio-economic one as well. Its impacts are felt unevenly across different communities, with marginalized groups often bearing the brunt of its adverse effects. Julian Brave Noisecat's insights into Indigenous communities vividly illustrate this point. Indigenous peoples, who often live in close harmony with nature, face unique challenges due to climate change. Their traditional ways of life and cultural heritage are at risk, making them particularly vulnerable to environmental changes.

These perspectives highlight the need for a comprehensive approach to tackling climate change. It requires not only technological and policy innovations but also an understanding of the cultural and social dimensions of the crisis. The differing impacts on various communities necessitate policies and solutions that are inclusive and equitable, ensuring that no group is disproportionately affected by the changes our planet is undergoing.

The multifaceted threat of climate change demands a response that is as complex as the problem itself. It calls for collective efforts that combine scientific advancements with socio-cultural understanding, ensuring a sustainable future for all. By acknowledging the diverse impacts and integrating various perspectives into our response, we can better navigate the challenges posed by this global crisis.

<img width="487" alt="Screen Shot 2024-01-14 at 1 49 47 PM" src="https://github.com/deholz/AreWeDoomed24/assets/87281088/7c8421eb-9459-41a8-b9dd-0adb5c8f62bb">

**Comment by timok15** [comment]
#climate, #framing, #salience

The uncanny valley is a phenomenon notably observed in the development of human facsimile robots. As the robot becomes more human-like and less machine-like, the robot becomes more likable to humans. However, as the robot approaches truly indistinguishable from a human status, its likability plummets. Presumably, though, if a robot could be constructed that was outwardly entirely visually indistinguishable from a human, a person would react to such a robot like any real human. This dip before full human appearance is the namesake uncanny valley. 

For the past several decades, multiple indicators of human well-being have been improving (number of wars, poverty, prevalence of diseases, etc.). Under conditions of a fossil fuels economy, the hateful aspects of human existence seemed to be retreating—agonizingly slowly, too slowly, but still retreating. Now the gains that fossil fuels have meant for the improvement in the quality of life of human beings have run out and instead their effects stand to heave us backwards. 

If we want to beat climate change (as best we can when even the best case scenarios are abysmal), human life conditions during climate change should be visualized as an uncanny valley-like curve. In the moment and for the coming decades, things are worsening and will continue to worsen (even just from what has already been released and the feedbacks tripped off, not considering any continuing uncompensated emissions until such and such date). Under such deteriorating conditions, Doomerism is easy to fall into thinking because everything, to all appearances, is pointing in that direction. However, with this conceptualization, we cannot and should not give up because we need to ensure that we humans do not make our home in the bottom of that valley. We must instead have our minds set on making it to the other side and its even higher slopes of human well-being. 

Even with the New Deal, it was not until WWII that the US economy really began to recover. In the absolute depths of the Great Depression, though there were various New Deal programs available to some, the good times were still years away. This is the presentist bias: it seems so bad, even inescapable, in the moment, but, once things really get better, the world seems so much brighter as if there was never in such a colossal and existential slump. I hold out hope that our descendants who will have the benefit of hindsight and of the complete story (of the century) will be able to observe our rise back out of this “quality of life valley” and connect our struggles with their own prosperity, whatever way it might look.

_(below) A depiction of the uncanny valley. Courtesy of Wikipedia._
![image](https://github.com/deholz/AreWeDoomed24/assets/155562196/f29052c9-e675-4ae8-bf3f-1f7dfc7b806a)

**Comment by lubaishao** [comment]
#origin
The breakthrough in climate science happened after World War II. In 1958, American scientists Charles Keeling and Roger Revelle established an atmospheric carbon dioxide monitoring lab at the summit of Mauna Loa volcano in Hawaii. The zigzagging upward curve provided by this monitoring station has become one of the most well-known images symbolizing anthropogenic climate change. In fact, disasters caused by climate changes had already occurred before that time. The mid-17th century is considered a "Little Ice Age." Many areas south of the Sahara in Africa suffered severe droughts between 1614 and 1619. In 1616, Japan faced the coldest spring of the 17th century. In the early 1640s, the East Asian region experienced crop failures due to abnormal weather. 
 
![a439ee16cacac15e977dc353e025c4e9](https://github.com/deholz/AreWeDoomed24/assets/155573451/31edcfef-0f47-4e8e-86ff-ea9f2b7b152e)

Until the 1980s, discussions about climate change were mainly limited to academic circles. However, the public began looking into the problem of climate change when the "hole" in the ozone layer appeared over the Antarctic region in 1986. Climate changes related to human activities become a common issue and an international political problem and movement. Since then, the United Nations has consistently urged the international community to reach consensus and take action on the issue of climate change but faced numerous challenges. Due to disagreements among some of the world's largest greenhouse gas-emitting countries, the Kyoto Protocol—a binding agreement demanding modest reductions in greenhouse gas emissions from wealthy nations—made little progress between 1995 and 2015. The Paris Agreement officially came into effect on November 4, 2016, becoming a legally binding international treaty. 
 
Even today, despite intense competition among major powers in politics, military, economy, and technology, climate issues remain one of the relatively easier aspects for them to reach consensus. I think it’s not only the significant economic implication keeping countries from making a consensus on climate change actions but also because it is a political issue. Politicians may be hesitant to commit to long-term initiatives for concerns of impacting their short-term public support. Simultaneously, there is a desire to benefit from improvements in the climate  environment committed by other countries through free-riding policies. However, major countries somehow have realized commitment to climate change is also a way to improve their soft power and ability to shape international order and norm. Moreover, political actions on climate change has turned from a political liability to a political bonus in the past decade.

**Comment by ldbauer1011** [comment]
#framing

Uncertainty of the future of the human species is nothing new. The apocalypse has been foretold ever since humanity could understand its significance as a species, with writings about the world ending being traced back to the cavemen. It’s logical to foresee an end, since every beginning is destined at some point to have an end. However, it is also easy to fall into the trap of letting the end of our story come to dominate the story itself. Such pessimism has a place in perceiving our world, and some pessimism is necessary to keep our actions grounded in reality. However, too much pessimism can lead to a lack of initiative and investigation into ways to shape our reality to solve the problem we are brooding about. Similarly, optimism needs to remain a factor when observing and investigating the problems of today. Without it, problem-solving is reduced to a moot point, since the inevitable outcome is the negative and deeply problematic one. Over-optimism on the other hand can begin to approach the fantastical, as our expectations begin to leave the limitations of reality. Both insights in moderation are needed viewpoints in order to effectively tackle the most dangerous short and long-term problems that humanity faces. 

When discussing problems that have unknown outcomes, fear becomes the dominant emotion quite easily. One of the most basic instincts within all of us is to survive, and it becomes impossible to position ourselves in a way that ensures our survival when the facts we are working with aren’t clear or understood. This instinct is extremely important, and the one thing I believe will save us. We as humans are an extremely diverse people, with a myriad of beliefs of how to live our lives, and what we need to prioritize in our actions. Survival of the species is something that is present in all of us, and inevitably the one thing that can serve to unite us. This desire to keep on living has a habit of steering us through dark times. I firmly believe that, when the chips are down, humanity’s collective instinct will allow us to maneuver through more dark times ahead. I admit that I am an optimist when it comes to the end of all things, but I believe that such optimism is needed to ensure we keep finding ways to solve problems. Without it, we may discover the end we are so anxious to avoid. 

![deec732dc575647d63b6a5706cdf70d3](https://github.com/deholz/AreWeDoomed24/assets/155675402/6f49648b-6e37-4628-9c65-c2274011589b)

**Comment by miansimmons** [comment]
#climate #framing #solutions

_This memo builds on the question I wrote for David Wallace-Wells: When it comes to human behavior, do you think that incentives and voluntary participation are more effective than punishment (ex: carbon tax) in motivating people to tackle climate change?_

Since many of the articles assigned this week were written by journalists, I began thinking more about the connotative language used to frame the climate change issue. Given that successful climate legislation in the U.S. is often incentive-based policy and not punitive, I tend to agree with Hannah Ritchie that positive framing is essential to motivating climate solutions. Negative messaging and threats can lead people to resort to blame shifting (e.g. "blame the boomers") or feel paralyzed to take action - and complacency will only make matters worse. In week one, Rachel Bronson acknowledged that the doomsday clock sounds inherently pessimistic and revealed that other names like the "life clock" have been considered to combat its negative connotation. In his writings, Wallace-Wells openly said he used to be a climate alarmist (which can be dangerous in the cult leader-types that spread misinformation and terror) and highlighted his optimism for the future. Negative rhetoric can lead people to dismiss claims as worst case scenario thinking.

An example of positive incentives working in practice is Wallace-Wells' assertion that renewable energy is attractive to nations when it is framed as an opportunity. A faster decarbonization path "could make the world trillions of dollars richer by 2050" (Wallace-Wells). China is manufacturing massive amounts of solar panels to distribute, for instance. They are making their country richer and expanding production/jobs, while signaling environmental consciousness to the rest of the world.

Meanwhile, in social science research, evidence is mounting that blaming and shaming people will not convince them to follow a set of rules or embrace a new idea. When people feel pressure to agree with something, their propensity for resistance increases. For instance, corporate employees often respond to mandatory training courses with anger and resistance, and many participants report more animosity toward the course material afterward (Harvard Business Review). However, voluntary training evokes the opposite sentiment and leads to better results (an opportunity for employees to embrace the training like it was "their idea" and signal dedication to managers). This illustrates that people often respond much better to positive incentives than punishment.

The key theme for this week is uncertainty. We all have to live with it, whether we like it or not. While negative messaging turns this uncertainty into something nefarious, positive messaging teaches us to embrace uncertainty as we strive to combat existential crises. It can be dangerous to wait for a certainty that will never come when enacting climate policy or engaging in decision-making. 
<img width="1067" alt="engagement scale " src="https://github.com/deholz/AreWeDoomed24/assets/155574830/adde71cb-2cb8-47b7-a178-ada730b2b606">

_How does your audience feel about your conclusion? Positive leads to engagement, while negative leads to burnout. - _LinkedIn, Change Management Journal__

**Comment by cbgravitt** [comment]
#solution #salience #framing #climate

The internet is an interesting place. Within it lies almost the entirety of human knowledge, and billions have easy access to it. However, there is much more on there than that. Any internet user can easily find a place to post their opinions publicly for all other users to read. Most of the time, these are under-moderated social media platforms who simply do not have the manpower or technological ability to ensure that every post lives up to their guidelines. This has led to the greatest source of information in human history doubling as the greatest source of misinformation. Looking through Facebook, Instagram, or Twitter/X, its almost trivial to find both avid climate change activists and firm climate change deniers. 

![image](https://github.com/deholz/AreWeDoomed24/assets/108561679/45a92d39-aa29-4b9f-a4b0-59d589d06c0d)
(From August 2023, Pew Research Center)

Climate change is one of the largest and most longstanding issues that has drawn a great deal of conspiracy theorists out of the woodwork to misinterpret legitimate data and spread faulty studies. The vast majority of the world is not used to parsing out and critically analyzing research papers, so many who see these misleading results don't question them. Indeed, with growing mistrust in world governments in recent years, very few areas of modern life are spared from conspiracy. From vaccines to climate to schools, people love to claim they are being lied to without ever claiming a proven truth. 

Fortunately, it doesn't seem to be the end of the world, pun intended. As some of the readings for this week noted, the world is still making progress against climate change despite a good portion of people simply not believing there's a problem. The problem is far from solved, but its not hopeless either. I can't help but think, though, that better and more efficient solutions could be adopted if only the public understood the problems better. Ensuring that the public understands these existential threats should be a priority in combatting them. Climate policy is a hot-button issue in modern politics. Ensuring the voter base is on the same page will propel positive policy and hopefully extend to educating them on other conspiracy-ridden fields.

**Comment by acarch** [comment]
#framing #salience - Even though climate change is unlikely to wipe out the human species, it remains an “existential threat” for two other reasons. First, climate change might not stamp out humanity, but it will put an end to certain peoples and nonhuman species (Ritchie, 2024: 13; NoiseCat, 2021). Island nations of the world have called climate change “genocide,” and African diplomats have called it “certain death” (Wallace-Wells, 2022). In the last forty years, over half of the world’s vertebrate animals have died (Wallace-Wells, 2019: 25-26). This wave of mass extinction is happening already, ending certain ways and forms of life forever. No matter one’s level of optimism, there is no arguing that climate change is the apocalypse—just maybe not for us and ours. 

Second, climate change might not risk humanity’s literal survival, but it certainly does threaten our existence. Whatever happens, we are not coming out the other side of this scenario the same. Climate change is an existential crisis that will permanently alter our identity as a species—maybe in ways we will find hard to endure. Our descendants will live in a burnt-out, flooded Earth rocked by pointless losses. They will be burdened by the labor of reacting to a problem they did not create. Climate change will transform not only our infrastructure, but our daily lifestyles, our social order, and the entire structure of geopolitics. As these problems worsen, we might find it impossibly hard not to hate each other. Andreas Malm suggests that a rational response to these problems might lie in acts of political violence, what many would call terrorism. Likewise, Amitav Ghosh suggests that catastrophe, such as a heat wave in Pakistan or India, might well lead to attacks against rich countries and their consulates (Wallace-Wells, 2022). This burden of resentment might inflame our already-significant urge to kill each other. And even if our species does not die, some of the survivors might wish they had. Climate change probably won’t kill every last one of us. But it is still a kind of “existential threat,” since it imperils the nature, if not the fact, of human existence. “Humanity” might still exist, but what it means could be entirely different.

<img width="622" alt="Screen Shot 2024-01-16 at 19 11 15" src="https://github.com/deholz/AreWeDoomed24/assets/155841186/57c3404e-a7f6-47b0-8204-062c38ef67af">

_Projected sea level rise threatens the Marshall Islands' [status as a nation](https://www.theguardian.com/world/2021/oct/17/rising-sea-levels-threaten-marshall-islands-status-as-a-nation-world-bank-report-warns)._ 

As a side note, I am also wondering if Ritchie’s feeling of being the lone optimist among pessimists and Wallace-Wells an alarmist among Pollyannaish optimists has anything to do with culture. Ritchie lives and works in the UK, land of the pessimists, and an island nation, which might make it a bit more vulnerable to the effects of warming. Wallace-Wells resides in the US, where people tend to value a more cheerful attitude. How much do these backgrounds affect their perspectives?

**Comment by WPDolan** [comment]
#framing #climate #nuclear #cyber

All of the existential risks we have discussed so far in class (climate, nuclear, and AI) can be attributed to major technological advancements. While developing an understanding of these underlying technologies allows the public to remain better informed of related existential risks, there are significant differences in our ability to model these individual risks.

Some existential risks, most notably those associated with climate change, can be extrapolated from existing datapoints and our understanding of ecology. Insights generated from these models can then be used to motivate (or at least try to, as David Wallace-Wells mentions in _The Uninhabitable Earth_!) climate action by elaborating upon potential consequences humanity faces if we fail to address their underlying issues. These predictions are possible largely because climate change is the result of many decades-long trends and because we have some understanding of how climate risks will be realized. As a result, the majority of climate change uncertainty is concerned with the magnitudes of predicted disasters. 

Magnitude estimates for the damages caused by nuclear risks also exist, many of which we read about two weeks ago in François Diaz-Maurin's article _Nowhere to Hide_. However, there is significant uncertainty regarding the likelihood of realizing nuclear risks in the first place; it is much more difficult to model the future political state of the world than it is to model global warming or ecological collapse. Motivating factors in employing nuclear weapons may vary heavily among individual decision makers within each nuclear power, many of which may arise suddenly and/or imperceptibly to the global community.

Modeling risks associated with AI is even more difficult. We have no history of interacting with advanced machine learning models, and no exact solutions to the alignment problem. Additionally, many machine learning models are "black boxes" in the sense that we are unable to obtain justifications for their decisions. Machine learning models may intentionally or unintentionally realize existential risks in a wide variety of ways. How would we sufficiently model and quantify these risks? Is it inevitable that any sufficiently advanced model becomes dangerously powerful so that they may better optimize their original goals?

Image: warming predictions for various emissions scenarios. 
![scenariotempgraph_0](https://github.com/deholz/AreWeDoomed24/assets/73420495/6b7f67d3-4163-4ea6-ad8b-150a84cb0c29)
Source: [IPCC, faq 12.1](https://www.ipcc.ch/site/assets/uploads/2020/05/WGI_AR5_FAQ_EN.pdf)

**Comment by lucyhorowitz** [comment]
#nuclear #origin #energy #climate
Our entire economic system is based around "growth." Whether you believe that capitalism is good or bad or should be maintained or town down, a complete revolution in the economic organization of society would have devastating consequences, at least in the short-term, and the economy is intimately tied to the amount of energy we can produce and consume. It is incredibly unlikely that we will ever meaningfully reduce the amount of energy we consume as a global society. In fact, it would probably be undesirable to do so—many people would suffer terrible fates if we were to mandate a reduction in energy consumption. High energy consumption is necessary if we want to maintain our food supply and even if we want to maintain feminist or egalitarian sensibilities (in many ways women were liberated from many forms of backbreaking housework with the advent of machines like dishwashers, washing machines and vacuum cleaners, all of which require large amounts of readily available electricity). It will be increasingly important if we want to realize any of the potential benefits of artificial intelligence. 
While renewables like wind and solar have made great strides in recent years, we still need to incorporate nuclear power in much larger quantities. It is reliable and creates massive amounts of energy while taking up minimal land and resources. Moving away from nuclear can only lead, as it has in Germany, to increased reliance on coal and other fossil fuels to provide that crucial "baseload" of power. 
The problem of nuclear waste is a localized one, and as such can be made into an acceptable cost when the alternative is global pollution and destruction of the climate. When done properly, nuclear waste storage is safe and robust and even represents a potential source of even more energy in the future.
It's a good thing that we live in an age of energy abundance, but without investment in nuclear energy it will be harder and harder to simultaneously hold on to the benefits of abundance and avoid the drawbacks of continually burning fossil fuels. Nuclear energy is the closest thing we have to a perfect compromise for modernity.
![5769e7c9243c1 image](https://github.com/deholz/AreWeDoomed24/assets/111657652/3af6e3f2-4c2a-4f1e-a55e-b031008ee658)

**Comment by miansimmons** [comment]
#movie #uncertainty #cyber #nuclear

The world is a complex system layered with uncertainty. When it comes to predicting the future and preparing for existential threats, social networks and state capacities will make the biggest difference in the face of uncertainty (Ferguson). However, leadership failures, misinformation, and irrationality present issues on this front, and often times ordinary people can matter more than authority figures in the event of a disaster. The movie _WarGames_ directed by John Badham captures the challenge of uncertainty brilliantly, addressing the potential for nuclear annihilation, rogue AI, and cybersecurity breaches (all of which we discussed in class).

The main protagonist of _WarGames_, David, is a high school student and technology whiz. He hacks into a government system called the WOPR, which is an AI created to run WWIII simulations and manage the deployment of nuclear weapons. Believing the simulations to be video games, David accidentally triggers a real thermonuclear war scenario between the U.S. and Russia. The WOPR then goes rogue and does everything in its power to accomplish its programmed objective - annihilate the United States. Despite the government refusing to listen to David and actively undermining any attempt to fix the situation, he is finally able to shut down the WOPR by teaching it that there is no winning side in nuclear warfare. 

The film shows how uncertainty can cause leadership to behave irrationally through failure of imagination and threat underestimation. The government could not fathom that a young man could hack into their systems alone and did not prepare safety mechanisms for their AI software. Therefore, they spread misinformation along their ranks and failed when disaster struck, assuming that David had been hired by the Russians and the WOPR was running perfectly. This movie illustrates that uncertainty is something we must all live with, and, rather than run from it, we must incorporate it into our models. Proactivity instead of reactivity is key. For instance, systems of communication/coordination, vulnerability assessments, and awareness campaigns must be established well in advance of crisis.

In terms of how realistic the presented risk is, we need only refer to Ronald Reagan's presidency. After watching _WarGames_, Reagan asked the chairman of the chief of staff, John Vessey, if what happened in the film could happen in real life. Vessey responded that the problem is much worse than he could imagine. Sure enough, in September of 1893 (three months after the movie came out), there was a false alarm nuclear warning that almost lead to weapons deployment by the Soviets.

Watching _WarGames_ definitely made the threat more significant to me, as there were many aspects of uncertainty and existential risk touched upon. The film's screenwriters interviewed prominent figures in computer programming and at the RAND Corporation to ensure the threat would be displayed as accurately as possible. The reviews echo my viewpoint. The movie grossed $125M worldwide against a $12M budget; people loved the discussion of emerging technology and continue to praise it today. A recent comment on Google Reviews states, "you know a movie is good if it can make people jump on their seats even after 4 decades!"

![WarGames movie poster](https://github.com/deholz/AreWeDoomed24/assets/155574830/c5e57190-9e61-4e52-81b8-2c6769793e62)

_WarGames movie poster, September 1983_

**Comment by M-Hallikainen** [comment]
#climate, #framing. #risk
During last week's discussion group session, the conversation turned to the doomsday clock and its recent incorporation of climate threat into its calculations. As we discussed at the opening of the class, the proliferation of nuclear weapons was in many ways humanity's introduction to the man-made existential threat. However, proliferation is somewhat exceptional in the rogue's gallery of existential threats in that there is a singular moment of decisive doom; the moment the missiles are launched and the world effectively ends; nuclear midnight. This decisive and singular end-state makes something like the doomsday clock uniquely effective as a tool to communicate the impending threat of proliferation, but this same strength becomes a major weakness when considering more progressive and multilayered threats like those of AI and climate change.

With the one-and-done apocalypse perspective typified by proliferation, progressive existential threats with uncertain futures and *relatively* slow developments tend to be pigeonholed in to one of two frameworks. Either the decisive final blow is an abstract concept far out on the horizon, one that we have ample time to correct from - even if only at the last second,  or its a critical deadline already passed - with no hope of recovering a now doomed planet. As David Wallace-Wells points out in the introduction to The Uninhabitable Earth, both of these frameworks tend to result in the same passive inaction spurred by willful ignorance or hopeless defeatism.

This specter of singular apocalyptic midnight has even crept into popular perceptions of existential threat models designed to illustrate their progressive and unfolding nature. As Hannah Ritchie points out, the IPCC's numerous climate targets are often misconstrued not as possible futures we should work towards, but critical thresholds that say "as soon as we pass 1.5°C, we're toast" (p.8). To an extent, I think this can even be seen in Geoffrey Hinton's argument in last weeks class that we are already past the point of no return with regard to AI development and that he takes some solace in knowing he is old enough he will likely dodge the seemingly inevitable consequences of already broken barriers. 

Our standing in the face of current and future existential threats is bad, and it is all but guaranteed to get worse. In many ways it is already impossible to recover the type of world we came from, and we will pass many more points of no return before things have a chance of getting better. I feel now more than ever there is a need communication tool like the doomsday clock, but not one that focuses on our creeping impending doom, but rather the progress being made, and more importantly the work still to be done.Something that stresses that things are getting worse, but that there is still a future to fight for. As Wallace-Wells points out, “the planet was brought to the brink of climate catastrophe within the lifetime of a single generation, the responsibility to avoid it belongs with a single generation,” (p.6) and as such one of the greatest dangers we could face is that of giving up.  
![image](https://github.com/deholz/AreWeDoomed24/assets/155584798/501f7923-4af6-4b54-9c90-3c606e054f0d)
_Image of a German climate protest in 2019, with the increasingly common slogan "You'll die of old age. I'll die of climate change". Captured by Boris Roessler_

**Comment by imilbauer** [comment]
#climate, #framing, #salience

When we are adapting to climate change, we must remember that there is more than technology to consider. We must also consider what technological imaginaries show us about how we feel about society.  Climate change is often framed as a problem that entraps us due to technological backwardness. The logic goes, our best defenses against climate change are new renewable energy technologies, smarter cities, electric vehicles, and even nuclear energy. The hyper-imaginary of climate change is one of constant innovation that leads us to dramatically transformed countrysides (solar panels and wind turbines galore), cities (plants growing from buildings, clean forms of transportation), and revitalized waterways and natural lands. New technology is critical but this imaginary has limitations. In essence, the technological futurism of climate change action paints a future both of a certain kind of vitality through revitalized nature but also one of a kind of asepsis. To be more precise, the futuristic cities we see mock-ups of are sites of asepsis aesthetically, because of their elimination of waste, their modern architecture, their tightly prescribed living patterns, and their lack of history. The modern city is a site of history and chaos because it is a site of politics, social life, and consumption. In New York, the heaping garbage, varied architectures, and confetti from New Year's Eve all speak to this complexity. Indeed, some consumption patterns hurt the environment. Some might argue consumption today is a symptom of capitalism gone astray. Yet, at the same time, it is symbolic of a kind of beautiful, messy, vitality in urban life which facilitates different patterns of living, styles of enjoyment, and a kind of multicultural social existence. Consumption demonstrates we are living stories. The vitality of nature, while beautiful, is different from the vitality of the modern city. While nature obviously isn't aseptic like the futuristic city, it resembles the vitality of the futuristic city because it is defined by order. Nature is more constant and understandable than history suggests that humans ever are. Whereas humans create stories, animals have lifecycles. So, while the technological futurism of climate change might be meant as a way to escape from global warming, this technological futurism may also create a spatial and social imaginary that escapes considering the realities of human life -- a reality with a unique vitality that is a mixture of beautiful, good, and concerning. A reality that can't be overlooked or erased.

As much as the beautiful and uniquely human aspects of modern urbanity necessitate consideration, the concerning aspects also necessitate consideration. These concerning aspects evidence the need for climate change strategies which consider the social and political challenges of addressing climate change. If society existed as a perfect body politic, perhaps, technological solutions would be the only strategy necessary. Since this is not the case, we must question how society's organization and structure affect our climate future. This might mean considering how inequality, gender discrimination, racism, and illiberalism might be barriers to climate action. It also might mean strategies to build storm walls and plan for the 200 million climate migrants David Wallace-Wells is predicting. 

In conclusion, I'm am drawn towards Hannah Ritchie proclamation of the need for "radical uncertainty" to deal with imagining the future in a clear-eyed manner. With too much optimism we can paint a one-dimensional ideas of society alongside our predictions of future technologies. As Ritchie discusses, with too much pessimism we can prevent action and fail to realize our agency in preventing global warming. With radical uncertainty, we can take the time to examine the theoretical priors that go into our efforts to adapt society to climate change in dramatic ways. In the grandeur of our desire to make adaptations, we can learn from bold idealists that came before us, like the Progressives, and realize that as much as technology can radically improve things, life's messiness must be considered. It should not nor cannot be wholly overcome. Adaptation will take the cooperation of complex human systems, like political organizations. Amidst the messiness, there is also beauty. Let's build smarter cities, but let's make them people friendly too. 

Photo:
![image](https://github.com/deholz/AreWeDoomed24/assets/63610244/685ade13-85df-4293-860c-e2099c53335c)
Saudi Arabia's future city "The Line"

**Comment by emersonlubke** [comment]
![frog in boiling water](https://github.com/deholz/AreWeDoomed24/assets/155669823/16ef3679-7224-45f4-b407-8dc63f2e62c9)
#origin #climate
     When thinking about how we've gotten to this point -- where the future of our climate, while not entirely bleak, is up in the air and it is seemingly well within the realm of possibility that we alter the environment to the point where it can no longer sustain us -- it's important to remember how piecemeal our collective progress towards the climate apocalypse has been. There have been individually disastrous technologies, from cars to plastics, but we are in the position we are for the same reason we are having so much trouble helping ourselves out: it is hard for individual actors to think that they are of legitimate consequence when it comes to the climate disaster. This can be seen on multiple levels.
     Firstly, individuals in societies generally don't critically think about their climate impact at the margins. There are many many people who engage in sustainable practices and I don't want to downplay the significance and importance of this, but most people just think along the lines of "I'm just one person, how big of an impact can I have," a dangerous mentality that promotes apathy and may well lead to disaster. 
     Secondly, larger scale actors, such as corporations and potentially nations as well, generally have their own best interests at heart. If a car company, for example, is worried just about their profit margins and thinks that spending on climate friendly technologies will set them back in the market, they can easily rationalize as "everyone else is killing the environment, why can't I" and the same problem as above ensues. 
      This sort of thing, the inability of individuals to feel the onus of saving the environment, reminds me of a sentiment that I picked up on in the DWW NYT article, that people think that everything is so marginal and 'small' actions will have no impact in the grand scheme of things. This leads to a society that is slowly marching towards a climate induced doom, where individuals think that they can keep kicking the can down the road, someone else will take care of it, it's not convenient for them to care about the climate so they won't, and we all slowly cook ourselves alive.

**Comment by timok15** [comment]
#movie, #framing, #environment, #rebirth

For this movie memo, I watched Nausicaä of the Valley of the Wind. Since I do not know Japanese and the version available in the Box file had no subtitles that I could find, I will be talking from the visual story, which will probably still contain spoilers, so beware.

The film opens dollying down the length of a tapestry scroll, cutting to scenes showing what the stylized art of the scroll is depicting. We are shown giant humanoids with glowing white staffs walking through burning and melted cities and enormous undulating beetle-things with several large eyes chasing the humans away from their old ways. Most of the land outside of the titular valley of the wind is devoid of any life, except for the fungal forest inhabited by otherworldly bugs (including the undulating beetles; a smaller one of these beetles can be seen on the poster below). 

Taking the whole film together, I see the film as warning against both the threat of devastating (nuclear) war and unchecked, unthinking industrial-technological development. The death giants brought the end of the world, but they are not machines, instead they are biology with pulsing flesh. The fact they looked like humans seemed to emphasize that we humans brought such death upon ourselves. Additionally, to make them be living, and not machines, also hints at these giants’ original creators messing with powers beyond their comprehension. 

In the wake of all the apocalypse, the fungal forest with its apparently hostile bugs springs up. The plants in this forest are infectious to pre-apocalyptic life, like humans and trees. However, despite the seeming hostility of both the wildlife and plants, the forest is in fact cleaning the water and soil, so that plants may grow again. To me, this last point illustrates the long term view thoughts that I have when I think about existential problems. Over the centuries and millennia following a catastrophe, things build themselves back up again, slowly but surely. It took millions of years for life to recover from the Chicxulub asteroid, but with that impact so far in the past now, the catastrophe time can be considered over, even if it is that impact which allowed mammals to seize out of the ash the crown the dinosaurs once wore. For humans, I would present the Bronze Age collapse. To be clear, I do not want a great calamity to befall humanity because such would be needless suffering. These observations are consolations, not justice for the destruction. 

“Consolation” is the one word I would use to describe this film. Hope is for the future where we solve the problems of today without civilization ending calamity and move on to a world which we cannot imagine anymore than even an emperor (someone who would be relatively more well-informed than the average person) in the year 1024 could have possibly imagined the year 2024. I surely prefer consolation to desolation where humanity is gone and has no more chances, but it is my sincere hope that the future is unbroken from our present.

_Theatrical release poster_
![image](https://github.com/deholz/AreWeDoomed24/assets/155562196/a7a77bcc-b257-4ae5-87a4-07537b4a0ac5)

**Comment by Hai1218** [comment]
David Wallace-Wells' "Beyond Catastrophe: A New Climate Reality Is Coming Into View" highlights a significant shift in our approach to climate change, emphasizing the evolving narratives that shape our responses. This shift is marked by technological advancements, successful commercialization of renewable technologies, and a realignment of global interests, steering us away from past simplistic and divisive narratives.

Historically, the climate change narrative was marred by an antagonistic view of emissions, framing the issue as a conflict between the Global North and South. Wallace-Wells notes a transition towards a more cooperative approach, where countries increasingly recognize shared responsibilities and the potential for joint action. We should continue to learn from this realignment, which transcended traditional geopolitical divides in the effort of combating climate change.

The narrative surrounding renewable energy is also evolving. Initially, renewables were seen as unequivocally beneficial, but I think this view is giving way to a more critical understanding that considers the environmental costs associated with their production. The successful commercialization of renewable technologies demands a comprehensive assessment that balances their benefits against their environmental footprints, ensuring true sustainability.

I think Wallace-Wells' perspective also challenges the narrative of climate passivism, which suggested a resigned acceptance of climate change. He proposes a new climate reality driven by proactive measures and technological innovation. However, he cautions that this technological optimism should not diminish the urgency of immediate and proactive climate policies. Technological advancements should complement, not replace, systemic changes in our approach to carbon emissions and environmental stewardship.

I do think we are not always failing at our goal to change a common narrative collectively for the greater good. The shift from using "global warming" to "climate change" in public discourse exemplifies the progress in shaping a collective narrative. This linguistic evolution, driven by collaboration among governments, scientific institutions, civil society, and the media, reflects a broader understanding of the climate crisis. It signifies a commitment to addressing the issue in a non-conflictual manner - going from a misleading, easily exploited narrative of "warming" to a more effectual description of the crisis. Wallace-Wells' insights pointed the importance of evolving beyond past narratives to meet the complex and interconnected challenges of our world.

![image](https://github.com/deholz/AreWeDoomed24/assets/89930221/7d0fceb6-d661-4ab4-b3c8-838fa584386a)
![image](https://github.com/deholz/AreWeDoomed24/assets/89930221/9d74efe0-e78c-4654-a40b-f88c90cd20aa)

**Comment by kallotey** [comment]
#origin #salience

Like I opened in my question, Wallace-Wells writes in Beyond Catastrophe that there is “suffering and injustice for hundreds of millions of people, because the benefits of industrial activity have accumulated in parts of the world that will also be spared the worst of its consequences.” This is unsurprising because we have seen thus far that climate refugees are coming from “developing” countries such as Somalia, Pakistan, India, and Nigeria because of flooding, drought, and other environmental disasters that cause displacement. Because developed countries have already reaped the benefits of earlier industrialization, they can step back and advise “developing” countries to reduce their carbon emission. However, as long as there remains issues with poverty, health care access, and food scarcity, industrialization in order to boost economies makes it difficult to see feasible compliance on this issue.

To capitalize on this uncertainty, the articles for this week also generally discussed an overwhelming sense doom that comes with the issue of climate change, especially in Ritchie’s introduction; she discusses how youth are very anxious about the future because of climate change. Something else that she emphasizes, however, is perspective. Last term, I took two classes: one that focused on the Blackness (Persons of Color) and Apocalypse and one that was specifically a human rights class. The image below is a production still from the documentary Reclamation: The Rise at Standing Rock I watched in the human rights class. Standing Rock is an Indian Reservation that lies on the border on North and South Dakota. The residents of Standing Rock had to fight against the Dakota Access Pipeline that wanted to tamper with their sacred ground. However, throughout the film, we can see that the youth of Standing Rock were more than willing to put their lives on the line to ensure that the construction would not be completed. And this example, similar to the examples that NoiseCat writes in his articles, are examples of how catastrophe and apocalypse for people of color look vastly different to that of their white counterparts (in this case, especially in the United States). Further, I would argue that this is even more an example of how strong uncertainty affects the lives of youth. The Standing Rock residents should not have to see themselves as sacrifices to get what they want which is to preserve their land, but they doubt the ability of their local government and federal government to support them and their wishes.

Given that climate change is perhaps not the most pressing issue going on for the Doomsday Clock, I, myself, am very uncertain about what future developments would look like. Moreover, I think the hypocrisy of the United States in particular when it comes to climate related issues such as environmental preservation makes me unconfident.

Source: https://reclamationthefilm.com/
![image](https://github.com/deholz/AreWeDoomed24/assets/156140764/65d26b91-c1d5-4647-a525-22e0d17fdfa1)

**Comment by summerliu1027** [comment]
#salience

I want to bring attention to another aspect of the challenges of discussing and acting on climate change, and that is the uneven distribution of wealth worldwide. Many regions, especially in developing countries, have not yet fully benefitted from the advantages of modern technology, which is often a precursor to understanding and addressing environmental issues. These areas are still grappling with basic needs and infrastructure development, making conversations about climate change seem very distant. A couple of years ago, I reunited with a few of my middle school friends in a small city in China, where we all grew up, and climate change was brought up in the conversation. I noticed a stark contrast in perspectives on climate change compared to those commonly held in America. One of them seemed like a climate change denialist. When I asked why he didn't think climate change is real, the response (directed at me) was: "You spend too much time in America. We here care more about making enough money and tomorrow's dinner."

The fact that some people live much closer to survival challenges renders the extinction narrative powerless in many scenarios. Such wealth discrepancy creates a divide in global climate discourse, where those who still struggle with securing basic necessities may perceive climate action as a luxury or a developed-world hypocrisy, rather than a necessity. Therefore, I think this disparity must be acknowledged and addressed in order to propel mobilization around the climate crisis. I think it is also fair to conclude that the failure to achieve global unison in a joint climate response is not merely a matter of ignorance or denial, but rather a reflection of the uneven socio-economic landscape that shapes people's and governments' immediate concerns and worldviews. The urgency that climate change discussions raise in wealthier nations is not easily transferrable to contexts where everyday survival is the primary focus. Therefore, global climate change talks should take on more nuanced approaches, possibly where tackling climate change is not a burden but an improvement of the quality of life and socio-economic development, even if this is more difficult to accomplish than simply dialing down emissions. 

![image](https://github.com/deholz/AreWeDoomed24/assets/155770194/c2c228b8-79fe-4e32-a49a-8a24428e3a54)

**Comment by maevemcguire** [comment]
#origin: 
Wallace Wells provides a comprehensive overview of the economic, technologic, and political modern history that led us to the current moment in climate change history. ‘Cascades’ in The Uninhabitable Earth: Life After Warming begins from the start, with the planet’s five complete extinctions and reaching the beginning of our modern understanding of climate change, with the establishment of the UN’s 1992 climate change frameworks.  
#risk: 
	Beyond Catastrophe: A New Climate Reality is Coming into View provides a nuanced,  and contemporary, and pressing yet optimistic view of climate change’s future and our role in preventing it. The “business as usual” continuation used to create panic of “civilizational collapse and even a sort of human endgame.” However, with recent progress from the past five years, expected warming has been cut in half. The question of whether or not we can fix the global ecological damages already done. However, he also discusses the risk of optimism becoming complacency, and the reality that “things will get worse before they even stabilize.” In The Uninhabitable Earth, Wallace Wells acknowledges that his conclusions and research is built upon much uncertainty in the form of human action and the climate’s reaction to this. He also discusses the economic risks associated with climate change – 2 degrees of warming costing $20 billion more than 1.5 degrees (page 27).
#solutions: 
	Wallace Wells also remains optimistic in his framing of our ability to help stop global warming. “Humanity retains an enormous amount of control.” Many of these solutions, he explains, come from advances in technological developments, including investment in green energy, decarbonization, cutting the prices of renewable energy, divesting from fossil fuels, a carbon tax, and expanding nuclear power. “We hold the growth story of the 21st century in our hands.”
#policies: 
	I also enjoyed his discussion of political/policy solutions and “climate justice,” including improving housing codes, sustainable urban planning, and improved water and infrastructure systems. However, questions of inequalities between powerful, emission-producing countries and Southern countries suffering, and predicted to suffer, the consequences of these emissions. Socio Ecological inequalities are predicted to rise, begging the question of reparations and the inability for some populations to ‘adapt,’ especially to inequities in energy access. He also discusses different political proposals to aid climate change, including the climate bill, and even the Green New Deal.
<img width="808" alt="Screenshot 2024-01-17 at 8 07 47 PM" src="https://github.com/deholz/AreWeDoomed24/assets/156890199/fe94015f-40d1-4d57-8432-96b449d14ba3">

**Comment by oliviaegross** [comment]
#risk #framing #solutions #salience

The impact of having proximity to the location and time period in which the effects of climate change are taking place has unclear consequences on the manner in which one will navigate these present challenges. I found Wallace-Well's arguments for both buoyant optimism and abject despair to be reflective of the multifaceted nature of the effects that global warming has on individuals. As he states, “...be more mindful of the inescapable challenge of uncertainty when it comes to projecting the future, and the necessity of nevertheless operating within it.”  I find this ‘necessity of operation’ that Wallace-Wells mentions to be unclear, especially as it relates to individuals' responsibilities in a world where uncertainty feels increasingly pertinent. As a young woman, one such responsibility that comes to mind is whether or not to carry a child. This was discussed in the Ritchie piece in relation to the seriousness of climate change. Ritchie mentioned studies which have shown a relation between being a non-environmentalist and being more likely to have children. However, she also states that, “I unequivocally reject, scientifically and personally, the notion that children are somehow doomed to an unhappy life.” (p. 7). How as women, we solve this tension, is unclear to me. Where do our individual liberties to feel wrong for creating life in this soon to be uninhabitable world end, and our responsibility to be a part of a solution for navigating this uninhabitable world begin? Wallace-Wells articulates that, “however intimidating it looks, that new world must be made navigable…” I find the testament of my female peers not desiring to have children understandable but simultaneously concerning. I find this question, of whether it is correct to bring children into this world to reflect the exact question at the core of Wallace Wells’s piece, to navigate the present challenge we find ourselves within, or not to navigate. As a woman, the existential threat from global warming (among many other threats) has been found in studies to keep women from having babies. To me, when thinking of how we navigate the present, this is a potential additional form of adaptation that was not discussed in the readings in this manner. While the forms of adaptation that were raised in Wallace Wells' piece were paths such as normalization, and movement on a massive scale, I wonder if one can additionally consider the adaptation of proactively not creating life to be an adaptation, or just a potential manner of intentional distinction. 


<img width="590" alt="Screenshot 2024-01-17 at 10 30 44 PM" src="https://github.com/deholz/AreWeDoomed24/assets/155587122/cb4490ed-c28f-432b-aa55-59f812e8a893">

**Comment by briannaliu** [comment]
**#climate #risk #salience**

We lie in a gray area for climate change, one that is better positioned than the 5-degree apocalypse forecast for this century but far worse than any sort of acceptable level of climate change. I fear that this gray area is actually more dangerous than the crisis mode we found ourselves in a few years ago, as it can lead to a false sense of security and a return to complacency. 

Crisis and visible consequences are what cause people to mobilize into action. Take COVID, for example. Before 2020, mRNA vaccines did not exist in the US. Yet when faced with COVID, the US was able to invent a vaccine within weeks and roll it out within months. This progress, which would have otherwise taken years, was accelerated in the face of crisis because COVID had very visible impacts on the US – it was killing tens of thousands of Americans. 

As morbid as it is, America doesn’t view climate change like we viewed COVID because Americans aren’t dying from climate change (yet). Climate change may not be killing people in the US, but it is having disastrous effects on other parts of the world. For example, [Kiribati](https://www.rescuethatfrog.com/before-our-eyes-the-sandbag-walls-of-kiribati/) is a Pacific nation comprising 33 islands and a population of about 110,000 that is expected to be largely submerged by the second half of this century. Despite the nation’s best efforts of building sandbag walls to combat flooding (see image below), Kiribati’s future as the first nation to be destroyed by climate change will be cemented by the carbon emissions of industrialized nations like the US. Meanwhile, in [Kenya](https://www.science.org/content/article/kenya-flooding-lakes-have-displaced-thousands-novel-lawsuit-blames-climate-change), its great lakes are flooding and displacing hundreds of thousands of people. Many towns are submerged, with only the tops of houses, hotels, churches, and schools still visible. Children in submerged schools can look out the window and see sea creatures like crocodiles swimming around their building. In [Indonesia](https://www.usaid.gov/climate/country-profiles/indonesia#:~:text=Around%20180%20million%20Indonesians%20living,life%2C%20property%2C%20and%20livelihoods.), around 180 million people living in coastal areas are vulnerable to rising sea levels and regularly face weather-related disasters that have killed and destroyed the livelihoods of many. Further, city-sized swarms of locusts brought about by climate change are devastating crops in [East Africa](https://www.nationalgeographic.com/science/article/locust-plague-climate-science-east-africa), where many people’s livelihoods depend on agriculture. 

In America we are much luckier. We haven’t experienced the fallout from climate change the same way these other countries have, leading to a western bias and a lack of urgency toward climate change despite being one of the world’s top polluters. In fact, America is so far removed from the visceral effects of climate change that it has become a talking point for some politicians to debate whether climate change is even real, or whether it is caused by humans. President Trump went as far as to pull the US out of the Paris Climate Change Accords, and his potential re-election does not bode well for climate action by the US.

Perhaps a crisis-mode mindset is necessary for America to meaningfully mobilize against climate change. 

<img width="701" alt="Screen Shot 2024-01-17 at 10 21 56 PM" src="https://github.com/deholz/AreWeDoomed24/assets/41585024/097e6cbe-436e-4695-8aa3-844d6dbfc508">

**Comment by aaron-wineberg02** [comment]
The text Not The End presents an image of young people giving up on the future. The polling offered by the text highlights a plurality, if not outright majority, of young people who are reluctant to be full members of their societies out of fear of doomsday. Much of this is framed around the increasingly concerning figures on climate change. To this point, many people have raised the alarm bell on the global sea temperature. One UN report that claimed many headlines noted that the world would face irreversible climate damage if the sea temperature exceeds a 1.5 degree celsius increase per year. This has been echoed by prominent politicians who influence young people including Congresswoman Ocasio-Cortez, a leading progressive figure in the United States. This is significant because these figures frame political attitudes and influence control social order. 

I appreciate that the text frames climate change from a repairable lens. Wallace-Wells provided a real contrast with his text in the NY Times. He made an emotional and compelling argument that human suffering will be immense without immediate climate action. This is a compelling human narrative but perhaps might not succeed with changing emissions from the ground up. I challenge this as potentially harmful to changing perspectives of polluters. Consider the solutions for climate change: transitioning to net zero in manufacturing, minimizing fossil fuel usage, and promoting renewable materials. All of these are issues that require cooperation with major corporate players. As Wallace-Wells suggests, Chinese manufacturers lie about emissions. Putting companies in the “bad guy” position and vilifying them may risk creating a shadow economy that creates polluters outside of society’s typical reach.

Rather, I propose an alternative: highlight sustainability as risk-proofing. Corporations are concerned about their longevity. As climate change becomes increasingly a part of corporate and political life, so comes with a risk of corporate irrelevance. New profits and incentives could be derived from the growing green economy. Meanwhile, oil and gas are temporary industries that could disappear in the coming decades. There is risk in maintaining a heavy exposure to revenue from pollutive spaces. This approach could work across the board. Rather, draw from Not the End and say that small contributions go a long way. Their presence in this market is not only helping restore confidence in our lifestyle but puts the company as a thought leader in a space many draw stress from.

#climate-change #policy #framing
<img width="935" alt="Screenshot 2024-01-17 at 11 10 38 PM" src="https://github.com/deholz/AreWeDoomed24/assets/156013714/ee184240-7c82-42cc-b32b-4e2861569eec">

**Comment by kallotey** [comment]
#novel

The novel I decided to pay particular attention to for this week was Fahrenheit 451 by Ray Bradbury. The novel paints a dystopian society where books are banned, and “firemen” are order by the government to burn books; books hold knowledge and some pieces of knowledge make people unhappy, so in order to ensure that people remain happy, the government censors it. The novel is also appropriately titled the temperature at which paper burns.

This week, the main existential challenge we read about is the impending doom of apocalypse as it relates to climate change. Fahrenheit 451 does not specifically dive into climate change per se, but it does in some ways tackle another one of the threats we acknowledge on the Doomsday clock: nuclear war. The book takes place in the U.S. where there have been two wars prior to the novel’s start, I believe, and the novel ends with an explosion which is likely a result of the ongoing war that citizens are completely unaware of. I would argue that censorship is not part of the existential challenges that we have or will be discussing in this class, so I can’t exactly say that this novel does a good job at addressing it. However, if we were to just look at it as an institution downplaying the severity of the war, I would say this is huge for the risk of nuclear war. Many Americans are pretty polarized on their concerns, or they deny nuclear warfare out of a psychological response to an issue far larger than them. In Fahrenheit 451, the government simply hides all of this information from its citizens, so they have no opinions and are able to live carefree. I suppose in some ways this makes things easier, it gives the government less to worry about if its citizens are so heavily monitored.

In terms of realism, I highly doubt America would go to the lengths it does in this novel regarding censorship. However, in the novel, the government does an insane job at propaganda in which it promotes entertainment. The main character’s wife has a huge screen TV in their parlor and is always watching TV. I think this is particularly concerning considering there is war ongoing in this society and she has no clue about it. This act is interesting because I think it quite literally happened a few months ago when the news of aliens started circulating and majority of people on my feed were questioning why the government thought we would care about this when we have other ongoing issues we would much rather prefer the government to address. I have to say, I am more concerned now because I wonder: could this be a tactic used in the future for damage control if the clock gets closer to midnight?

Source: https://images.app.goo.gl/aXPgxb1N4foAxYSp9
![image](https://github.com/deholz/AreWeDoomed24/assets/156140764/59910881-8971-4230-abe7-1593fe5c62c6)

**Comment by madsnewton** [comment]
#framing #climate

Not the End of the World by Hannah Ritchie had a fairly positive outlook on climate change. It was an interesting and refreshing viewpoint to read, considering most things I read about climate change and the environment are super depressing. Specifically, I liked the Venn diagram under section 3 of the “Six things to keep in mind”. Ritchie takes a quote from her colleague “The world is much better; the world is still awful; the world can do much better” into a Venn diagram to represent that all three of these things are simultaneously true. 

Especially for a class titled Are we doomed? Confronting the end of the world, I think this is a good perspective to have on the climate crisis. Doing any kind of environmental work can start to feel meaningless when the headline of every environment/climate change article threatens human extinction in X years if we don’t do this, this, and that. I do think it is very important to be sounding climate alarms, but the good stuff needs to be there, too. Or else, climate change deniers and skeptics that are drawn to the big, clickbait headlines, will only ever see the bad and will think that there is nothing humans can do to reverse anything. 

I think that the response and success with the ozone layer is an example of good media coverage regarding an environmental issue. The Montreal Protocol and Kigali Amendment successfully regulated CFCs and HFCs and now the hole in the ozone layer is predicted to be recovered by 2040. Some countries still have high levels of CFC and/or HFC emissions, but the ozone layer is still recovering! Hearing this news makes people feel more optimistic about the environment and shows that change is possible when it comes to this stuff.  Overall, I think that the response to climate change should be less climate nihilism, and more climate optimism. Climate change is awful and scary, but progress is still happening and that can’t be ignored in favor of doomsday headlines. 

![ozone_recovery_update_2017_print](https://github.com/deholz/AreWeDoomed24/assets/155592954/3dcc01ff-7f15-485e-8b6c-5feddd10047f)
_Ozone layer recovery: rare environment success story_

**Comment by gabrielmoos** [comment]
#framing, #climate, #emerging

In Wallace-Wells’ NYT article, he briefly mentioned the emergence of BRICS, but only in the sense that they are developing a Lithium OPEC. I’m curious to explore how the growing fears and pains around climate change will change the world order from a Western hegemonic structure towards a BRICS group. In a sense, do the effects of climate change mean an end to Western hegemony, and that power shift towards population-increasing and resource-heavy countries, as opposed to wealthy ones with stagnating and declining populations? 

Climate change is not only front of mind but pivotal to the livelihoods of BRICS citizens, through massive amounts of riverbed flooding or increased heat/droughts near the equator. Western countries are globally further north leading to more severe weather, but can avoid these short-term devastating impacts. This likely means that most climate innovation and future technology will stem from countries attacking the problem head-on, which is supported by the development of the BRICS bank, which has a goal of financing clean energy and sustainable development as BRICS countries are still expanding their energy and physical infrastructures. Birth rates in BRICS nations have dropped considerably over the past 20 years, however, still above replacement, contributing to the coalition’s massive economic growth. This economic growth and ableness of the population, positions BRICS nations with opportunity and labor resources to tackle the problems of climate change, whereas Western nations have seen declining birth rates and are further separated from the immediate effects of climate change, thus leading to less promising innovation in the space from the West. The need to live will always overcome altruism.

These statements fall in line with economic data, which predicts BRICS nations’ GDP are expected to grow by 189% by 2050, whereas G7 countries only expect 50% (According to Goldman Sachs). This discrepancy highlights the economic shift before our eyes in the most imminent moments of climate change. Does this mean that we’ll all be speaking Mandarin in 30 years, probably not, but this might mean more trade occurs in RMB or Rupees as opposed to solely the dollar. We already know that the West is facing pressure from BRICS nations regarding currency hegemony, production output, abundance of cheap labor, and economic growth. The West still holds a lead regarding innovation (according to the World Intellectual Property Organization), however, BRICS nations are not far behind. I think climate change and the goal of 2 or 4 degrees Celsius, will be the catalyst for the shift of the global hegemonic structure from the West to BRICS.

![BRICS vs USA](https://github.com/deholz/AreWeDoomed24/assets/91801383/562cbc1e-9d7e-4db3-9fb2-56cdf27c8e27)

**Comment by mibr4601** [comment]
#solution #climate
Many individuals currently perceive climate change as beyond their control, and when they become parents, this perspective often gets passed on to their children. This can lead to a sense of being born into a seemingly bleak situation with no power to effect change. The widespread adoption of such a viewpoint in our culture creates a barrier to public engagement in addressing climate issues. Additionally, children's education often lacks sufficient emphasis on the dangers of climate change and other societal threats. In my early education, for instance, the topic of climate change was notably absent, despite living in a state renowned for its natural beauty—Colorado. Most people did believe that climate change was a real threat, yet most people did not take any action and rarely voiced their concerns among friends. Despite it being a subject so relevant, no one wanted to talk about it, especially not teachers. While you can't force anyone to take a stand against climate change, avoiding discussions about it certainly won't contribute to fostering a sense of concern or engagement among people.

So what exactly needs to change from an education standpoint to ensure that we are having positive conversations? To start, I think that there should be more of a focus in earlier education on informing students about how the climate is changing. This could be through having talks that are solely focused on climate change or even through connecting other materials to climate change. It should not solely be about whether we are doomed or not, but more on how things are changing and how things could look in the coming years. For some people, the imperative sense of doom might make them want to take action whereas for others, it might just leave them feeling helpless. On the opposite end, you don’t want people to think that the problem is smaller than it actually is. I think that it's important that people believe that there is still hope and that their actions can cause change whether that is through minimizing their emissions or through lobbying for environmental legislation. If students talk more about climate change whether that be with their parents or with other students then they may be more likely to speak up and take a stand. I believe that it is necessary that there are protests against big companies that simply profit maximize without a care for the environment. This is not only against companies but also against the energy wasted by the government. Thus, I think it is critical to have more discussion at an earlier age that will hopefully lead to more students caring and taking action. This comes from depicting the real-world implications of climate change and how their day-to-day life is going to change. This might be part of the reason that there isn’t much talk in the classroom.  When students are made aware of how their individual lives are going to be worsened by climate change, then more students will want to do something about it. For Colorado, I am sure that if there was a proven threat to skiing in the next few years, then students as well as adults would start to make a ruckus.

![image](https://github.com/deholz/AreWeDoomed24/assets/114629715/9a6a2150-ee05-436d-8b25-36c827ff5a72)

**Comment by agupta818** [comment]
"Questions about the future course of coal had been circulating for years, often raised by the same people who would point out that projections for renewable energy kept also comically underestimating the growth of wind and solar power." This is from the NYT article that was assigned to this week and after reading this, I began to think about how we can turn to alternate power sources at a greater level than before and what these alternatives look like. While solar, wind, and hydro have already been integrated into some power systems, they have not become leading sources of power even though they are so much greener than the alternative. They are unfortunately still expensive to implement, and, as the article states, people underestimate renewable energy sources and how much we can expand them.  In October, the Biden-Harris administration announced $30 million dollars dedicated to clean energy funding in 28 states. Yet, is this enough of an effort from the government? With billions of dollars being invested in other sectors, $30 million seems far too little. In fact, it seems like for most administrations, this has not been a priority because they believe the doom we will face from the climate is far off and that there are more imminent threats, or that in their lifetime, this is not something that will effect them. Yet, we already see the impact on natural disaster frequencies from adverse effects of climate change, such as the  rise in sea levels, in many countries around the world. What is worse is the inequitable effects of carbon footprints of developed countries on developing ones. This is why governments like ours need to increase funding for clean energy and climate research into these solutions that matches the impact we have on climate change. These solutions could be optimizing our wind, solar, and hydropower systems, or researching alternate solutions. For example, a recent news article I came across was one which I had to read the title of twice! BBC published an article on an aviation company that developed jet fuel made from human sewage. Tests by international aviation regulators found it to be almost identical to the standard jet fuel. While there is great promise in their results, is it a solution for other industries outside of aviation?While I may not know the answer to that, increasing funding and awareness of work like this can only help us improve what the future of clean energy looks like. I remain hopeful that we will be able to have a future that uses more clean energy than fossil fuels, when that will be, I have no idea!

#solutions #policy

<img width="755" alt="image" src="https://github.com/deholz/AreWeDoomed24/assets/150064153/7ae823c6-4edc-43a3-9185-c0c5a3aca982">

**Comment by AudreyPScott** [comment]
#framing #climate

In free time before my discussion session this week, my group and I discussed the framing of climate change as a risk, debated the applicability of a clock-like symbol, and discussed the role of optimism and avoiding doom-and-gloom ideology in addressing the climate crisis. In reading these texts, I've found myself still feeling a bit hollow when thinking about the topic: in Uninhabitable Earth (or at least our excerpts), end-of-times philosophy is discussed alongside tangible impacts of climate change. However, in the New York Times article, Wallace-Wells takes a more optimistic approach, emphasizing -- perhaps neededly -- our ability to change through both his words and those of leading climate scientists. However, I find it incredibly hard to stay optimistic -- I can read in these articles about the paper-pledges of companies and governments, and how that's maybe a plus, or how having communicators like Greta Thunberg are good for the movement. I can see climate scientists, working with their models and theories, say that things are still bad, but workable -- we can still turn around (a sentiment also echoed in Not the End of the World). 

I know, of course, that this line of thought is necessary. Saying there's no way to turn back won't solve anything, and it may only spur some towards inaction. In that sense, I agree more with the pessimistic, drastic tone taken in the Uninhabitable Earth (even though I didn't much appreciate the presentation of the information). However, when I was a sophomore in high school, I lived through one of the events cited in Wallace-Wells' book. Hurricane Harvey (I'm using an e-book so page numbers aren't reliable, but in Cascades) wreaked havoc on my community. The ones who were safest were those whose neighborhoods were built post-Katrina, but that didn't help the dozens who died of direct fatalities, or the plenty more who lost their homes for a long while. I volunteered locally, organizing and distributing supplies at houses of worship and collecting funds for on-the-ground need response. I saw the shaky hands of those who lost everything; I see the tension that comes in peoples' necks when a storm comes down a bit too hard. If the error in climate change communication had been for decades the setting and pushing back of targets, of instilling fear without action, an error I see rising is the readiness to divert to models and theory -- tools of the ivory tower that don't capture the practical realities of pain and suffering. If my experience with Harvey affected me so, I can't even imagine what was going through the heads of the 45 million displaced in South Asia as he then goes on to discuss. I don't know how these practical concerns fit in; whether it's crisis response and being able to quickly attach these issues as they're happening or letting things simmer out. But I sure don't have much of a capacity to feel hopeful when I'm 50 feet above sea level and 50 miles from the coast.
<img width="839" alt="Screenshot 2024-01-18 at 12 01 31 AM" src="https://github.com/deholz/AreWeDoomed24/assets/89955782/ec49891b-fbbc-4ecb-bc83-ffa3ec61d69e">
^ Me and a kid from my neighborhood (should be close to finishing high school now!) being tasked with water distribution at a supply collection site. He had a handle on things.

**Comment by gpafentakis** [comment]
#climate #origin
Annually, many people experience the impacts of global warming first-hand, transforming what was once considered a theory of the future into a present-day reality. Unraveling the origins of this phenomenon and understanding how international agreements have shaped it allows us to contextualize the situation and make more informed predictions about the future.

Joseph Fourier provided one of the earliest descriptions of the greenhouse effect in 1824, but it remained a hypothesis, marking just the beginning of extensive climate research. The awareness of global warming and the understanding of the underlying physics gradually grew, reaching a consensus among scientists in the late 70s and early 80s, who then began warning about its potential dangers. Ice core analyses during that period revealed a significant increase in CO2 linked to human activities since the Industrial Revolution, emphasizing the urgent need to curb greenhouse gas emissions.

Then, the world started taking precautions. In the graph below, we can see the long road that resulted in the current climate agreements. Governments began international collaboration after the first UN conference on the environment in 1972. Subsequently, the Earth Summit in Rio de Janeiro in 1992 marked a significant milestone where industrialized nations pledged to reduce greenhouse emissions to 1990 levels, though the agreement was non-legally binding. The first legally binding agreement materialized in 1997 with the Kyoto Protocol, in which economically developed nations committed to cutting emissions 5.2% below 1990 levels by 2012. In 2007, under the "Bali Road Map," governments committed to a two-year process aimed at achieving a binding agreement at the 2009 UN Climate Change conference in Denmark. Unfortunately, these negotiations proved unsuccessful, leading to the absence of a new treaty. In a new effort towards a global plan against climate change, governments made a second attempt through "The Platform for Enhanced Action" in 2011, initiating a process to secure a binding agreement by 2015. Finally, in 2015, this highly sought-after agreement was achieved through the Paris Accords. Countries agreed to limit global warming well below the 2℃ threshold and strive for 1.5℃ if possible.

![a-history-of-climate-change-negotiations-1461344627368](https://github.com/deholz/AreWeDoomed24/assets/155593692/988b5cc6-70cf-41e0-afd4-cf6c0e84adb1)

David Wallace-Wells in his article, "Beyond Catastrophe, A New Climate Reality Is Coming Into View," highlights that initial predictions for global warming were dire. However, the lengthy negotiation process and the resulting Paris Agreement have improved the outlook for the future, making the doomsday scenario seem improbable. Wallace-Wells acknowledges that despite this progress, humanity still faces highly disruptive consequences like floods and wildfires. While the need for a more robust international agreement to mitigate risks remains evident, the historical review underscores the slow pace at which governments operate. Despite nearly two centuries since the beginning of global warming research, legally binding agreements have been in place for only around 30 years. Considering the challenges and the extensive negotiations behind these agreements, the probability of a more binding one seems small. Therefore, focusing on policies to manage the consequences of global warming in the near future may be a more pragmatic approach that can safeguard human lives.

**Comment by AnikSingh1** [comment]
#climate #framing #solutions 

The biggest claim of division I see involving this problem of climate change is social unity. As a result of high carbon in the atmosphere, the world has an inevitable problem that only makes the situation worse. In order to solve this problem, it is clear that awareness needs to be made to see results. However, this is only through the political lens where problems are made apparent - personally, no individual person feels as though they are contributing to the scale at large. It is this distinction of unity between the individual and the public that I believe is the true social dynamic which hurts climate change, the atmosphere, and our world. 

One of the largest contributing problems with climate change and the overall atmosphere is actually derived from human technology and growth. With good human intentions, humans have sought to use new developments to create a better world for our livelihoods day by day. It is through these new methods where we have advanced as a civilization to improve and see positive benefits to our lives. As a side effect of this production, the increased carbon emission has led to a significant issue for political and environmental economies. But because these innovations were made with positivity and growth in mind, they couldn’t possibly be bad in these realms - at least, that’s what was initially thought. Because humans are not provoked to immediately see a problem with their actions, the incentive to protect the environment around us is diminished - all because of one main thing: consequences. These consequences, as delayed as they are, do not promote a sense of urgency or immediacy until a person feels an impact. And this impact could be politically/socially, involving peer pressure to hold each other accountable for positive change. It is through these dynamics of human communities where I see legitimate chances for improvement against this problem for the future. Bringing awareness to improve the situation is definitely beneficial, but the immediacy of it all will never spur any change unless there is an immediate, coming consequence in the near future. I hope that focusing on some kind of policy or control union to force change to be mandated could save lives for the future generations to come - because as of right now, the clock is ticking and opportunities are being lost quickly.


![climatecomic](https://github.com/deholz/AreWeDoomed24/assets/155674693/aab2b15a-866c-48d7-8daa-f8e946ec357a)

Above comic involving a rich fashioned man not seeing immediate problems - they do not exist because they are not happening immediately, right? 😨

**Comment by Daniela-miaut** [comment]
#origin #solutions
I see the climate issue as more of a problem of responsibility, than of technology. For example, I somehow feel that denying climate change has become a display of masculinity. It endorses the exploitation of nature (together with the right to survival of others), in a disguise which falsely render the cost of their abuse as “inevitable”. The observation might seem weird, but I think some politicians (including some heads of states and governments) are utilizing their denialist attitudes (as well as other acts that show their masculinity) as a way of consolidating their stance against the dominant (and maybe middle-class) western values.
Although climate issue has aroused all kinds of anxieties, including the fear of apocalypse and the fear of having fear of it, it seems that we have not yet apprehended the consequence of climate change in a tangible way. Both the fact itself and its future consequences are not very intuitive. Also, while privileged people think they can get away, underprivileged ones are preoccupied with other urgent survival challenges, or are simply silenced. I would argue that despite the heated discussion, not so many people truly care about the possible catastrophe itself.
This diagram shows a few facts (google translated from Portuguese) about Amazon rainforest fires. Those forest fires are not caused by natural reasons, but by intentional burning of the forest to clear land for agribusiness (encouraged, or at least tacitly consented by governmental policies). The capital gain profit from the expansion of their business, while the costs are burdened on people who are most vulnerable to climate change (and especially, people who live in the rainforest, including a lot of indigenous people who are forced to leave their land). Aside from being another piece of evidence of the severity of climate issues, I think the data also remind us to what extent privileged people can ignore the risks and disasters they bring to others.
<img width="1179" alt="Screenshot 2024-01-18 at 6 53 39 AM" src="https://github.com/deholz/AreWeDoomed24/assets/143459193/d297e6b3-3a2d-421c-b284-4001dc591ed4">

**Comment by tosinOO** [comment]
#policy #solutions

The Harper's Magazine article provided, I thought, an in-depth examination of how climate change narratives have changed, especially within media coverage. From dire predictions to nuanced understandings, media portrayals have changed as we as a society adjust to climate change - mirroring its affect and response mechanisms over time. Examining this change provides invaluable insight into individual and collective actions and policies which could address its risks more effectively - particularly regarding renewable energy as a solution.

Renewable energy was once considered a peripheral element in the larger energy discussion. I'd argue now it has moved to the forefront. This shift is driven by both technological development and public perception; historically speaking, fossil fuel-dominated policies were driven by economic and political interests; transitioning towards renewables may be seen as analogous to drastic public health measures during plague outbreaks when society faced similar existential threats, like quarantining them off in response. 

I think renewable energy's impact can best be understood when we consider its cost-effectiveness and scalability - factors which have seen significant progress over the past decade. Not only have technological advances advanced significantly over time but policy interventions like subsidies, tax incentives and research and development funding have played a crucial role in making renewables practical solutions. Still, its implementation does present unique challenges such as significant infrastructure changes required and socio-economic effects on communities reliant on traditional energy sectors. 

Disagreements seem to often center around the pace and feasibility of this transition. Critics frequently point to limitations inherent in renewable energy technologies like storage capacities and intermittent supply. But such challenges should not be seen as insurmountable; rather, they point towards areas that require further innovation or policy support.

Looking forward, individual actions such as reducing energy consumption and supporting sustainable practices cannot be overemphasized. Collective initiatives, like community-driven renewable energy projects and public advocacy for green policies are equally vital; together these measures, along with technological innovation and policy reform efforts can significantly lower climate risk.

As Harper's Magazine points out, renewable energy's transition is an integral part of our response to climate change, showing how shifting policy, technology, and public perception can all come together to address existential risks. While challenging, this transition represents a tangible pathway toward mitigating its impacts while drawing parallels to previous responses to crises while emphasizing both individual and collective actions as possible solutions.
![how-to-reduce-global-warming-1203897-v2-8f28da468bc8402fbe6a359e519d620b](https://github.com/deholz/AreWeDoomed24/assets/155605430/82c2930e-af51-456d-85f6-a3a9dca7e757)

**Comment by oliviaegross** [comment]
#movie #uncertainty #cyber

Throughout the quarter, we have been assessing what appears to be the present pivotal point in human history, in which civilization is facing unprecedented threats. The movie _The Circle_, directed by James Ponsoldt and released in April, 2017, particularly emphasizes the potential for, and existence of, self-inflicted human catastrophe. The Circle portrays a fictional Silicon Valley with timely themes related to technology, social media companies, and the existential challenges they present.

The protagonist, Mae, is a new employee at The Circle that is performing her job well. She is handpicked by the co-founders of the company to take part in an experiment with a new small camera they've invented. Mae has agreed to wear this technology on herself all the time and embrace the idea of "total transparency" and complete “integration” with technology. The movie focuses on themes related to privacy, surveillance, and accountability. The plot of the movie invites reflection on the role of companies like Google and Apple in our everyday lives and the manner in which they continue to push the boundaries of privacy and the existential challenge they present. More particularly, Mae in her participation in the trial for this product, stands as an example of someone who makes great sacrifices for the advancement of technology. However, the film’s plot emphasizes how the founders of the Circle are not willing to live in such “total transparency” and sacrifice personal freedoms in the same way others, like Mae, do. This tension in the film brings attention to the fact that not everyone is willing to have their privacy violated and raises questions related to what individuals, especially those in power, are willing to sacrifice in the name of technological advancement. 

The film made me reflect on the balance to strike between privacy and transparency. In "The Circle," the theme of complete transparency is explored simultaneously with the catastrophes that arise from an overly transparent society. Furthermore, this film invites discussion on user’s lack of awareness of the manners in which their privacy is being infringed on. This brings attention to the current existential crisis of our liberties being infringed and our simultaneous extreme dependency and frankly at times, addiction, to technology. _The Circle_ realistically reflects the risks posed by technology companies by way of its access to users' data, ability to monitor individuals' lives, and thus ability to sell new products and resell information to third parties.

Additionally, in the trial Mae participates in, the threats of embracing cutting-edge technologies without proper consideration for consequences was mindfully explored. In _The Circle_, the risks presented by technology are not only realistic, but are currently taking place, just open up your phone and read the privacy agreements we consistently both skim through, and agree to. After watching _The Circle_, I feel I have been reminded of the manner in which we are all constantly and carelessly, signing our personal freedoms away. The movie stands as a productive exercise, as there were aspects of technology related to uncertainty and existential risk that were touched upon that I do not often reflect on. However, while I found the movie effective in raising pertinent questions related to the existential challenges technology invites, it received a 16% on Rotten Tomatoes and a 5.4/10 on IMDb. However, many of the Google reviews disagreed. As one stated, “The Circle is for its most part a timely and somewhat disturbing look at our lust for technology and the loss of privacy in our world today and where we are all heading at a frightening rate.”

<img width="376" alt="Screenshot 2024-01-21 at 9 28 01 PM" src="https://github.com/deholz/AreWeDoomed24/assets/155587122/d9e7f16d-5798-4931-a2dd-3751623fa298">

_The Circle_ Movie Poster from April, 2017

**Comment by ghagle** [comment]
**#salience**

Wallace-Wells, in _The Uninhabitable Earth_, briefly alludes to what I think is one of the most crippling threats facing the planet with respect to climate change: the international struggle for economic and political might and independence. He notes how "if you had to invent a threat grand enough" to usher in "true international cooperation, climate change would be it" (Wallace-Wells 25). Yet, he continues, "we are only unbuilding those alliances ... and retreating from collective responsibility" (25). Paired with Ritchie's focus on mentality as we strive to tackle climate problems, his truism takes on substantially more weight. After all, the international 'retreat from responsibility' does a lot more than lead to the pumping of more fossil carbon into the atmosphere. It also undermines many of Ritchie's enumerated assertions for optimism that she does not address. In particular, because of the powerlessness of the international order and the 'retreat' that powerlessness enables, the extent to which we operate "alone in this" struggle and the "afford[ability of] complacen[cy]" are both exacerbated in ways that cripple humanity's optimistic mentality towards the environmental crisis and, therefore, our worldwide response to it (Ritchie 15-16). 

Because the people, interest groups, and government of one country have little to no influence over another country's climate policy, the a sense of independence exacerbates an aura of futility in efforts to make changes, especially when other countries' effort fail to match one's own. Ritchie asks us to trust in the fact that "there are _many_ people who are working to build a better future" (16). While there are, the political barriers that separate those who are working on this future confine their voices and their power. Climate change is a global phenomenon with global sources. Almost all public efforts to address the issue do so in one or a handful of countries. Comprehensive, global reform remains out of reach. For instance, the 2015 Paris Climate Accords is an agreement, not a reform. It is not a world-encompassing law and is not, therefore, above the isolation of policy-making that contemporary borders create. See no further than the success of countries to abide by their own agreement. The sense of futility that the international system contributes to therefore essentially encourages the inability of countries, companies, and individuals to adequately address the needs of the climate. Excuses become defendable and behaving without environmental constraints becomes advantageous. Other counties aren't contributing their fair share. Why should we? one might ask.


![image](https://github.com/deholz/AreWeDoomed24/assets/114632677/b152fd5c-dd6b-4486-852d-69227fdc7654)
_Efforts to address our climate destiny are in global 'retreat'_
